cloudsync/sync/manager.py

```diff
@@ -1388,6 +1394,25 @@ def embrace_change(self, sync, changed, synced):  # pylint: disable=too-many-ret
         log.debug("nothing changed %s", sync)
         return FINISHED
 
+    def handle_changed_is_missing(self, sync, changed, synced):     # pylint: disable=no-self-use
+        if (sync[synced].exists == EXISTS and sync.paths_match(changed) and
+            sync[changed].hash == sync[changed].sync_hash):
+            #TODO: and not sync[synced].changed -- another condition that was in Mike's original change
+            if sync.priority <= 2:
+                log.warning("%s missing, other side exists. punting: %s", sync[changed].path, sync)
```

Comment:
we an throw the changed time advancement if we want the punt to be more aggressive.  in general, that's the only time the change time is valid.

Reply:
increased punt limit to 5